File: Assignment_13/Pattern_1.c
///////////////////////////////////////////////////////
//
//  File name :     Pattern_1
//  Description:    Accept number of rows and columns from user and display pattern. 
//  Author :        Trupti Bhatlekar
//  Date :          25/05/2025
//
///////////////////////////////////////////////////////
// Input: 4 4 
// Output: A B C D
//         A B C D
//         A B C D
//         A B C D          
///////////////////////////////////////////////////////

#include <stdio.h>

void Pattern(int iRow, int iCol)
{
  int i = 0;
  int j = 0;
  char ch = '\0';

  for (i = 0; i < iRow; i++)
  {
    ch = 'A';

    for (j = 0; j < iCol; j++)
    {
      printf("%c\t", ch);
      ch++;
    }
    printf("\n");
  }
}

int main()
{
  int iValue1 = 0; 
  int iValue2 = 0; 

  printf("Enter number of rows and columns:\n");
  scanf("%d %d", &iValue1, &iValue2);

  Pattern(iValue1, iValue2); 
  return 0;
}

File: Assignment_50/Pattern_1.java
///////////////////////////////////////////////////////
//
//  File name :     Pattern_1
//  Description:    To accept row and column from user and display pattern.
//  Author :        Trupti Bhatlekar
//  Date :          29/08/2025
//
///////////////////////////////////////////////////////

//////////////////////////////////////////////////////
// Input: iRow = 4; iCol = 4
// Output: * * * #
//         * * # *   
//         * # * *
//         # * * *  
///////////////////////////////////////////////////////

import java.util.*;

class Pattern
{
    public void Pattern(int iRow, int iCol)
    {
        for (int i = 0; i < iRow; i++) 
        {
            for (int j = 0; j < iCol; j++) 
            {
                if (j == iCol - i - 1)
                {
                    System.out.print("# ");
                }
                    
                else
                {
                    System.out.print("* ");
                }
                    
            }
            System.out.println();
        }
    }
}

class Pattern_1
{
    public static void main (String A[])
    {
        Scanner sc = new Scanner(System.in);

        System.out.print("Enter number of Rows: ");
        int iRow = sc.nextInt();

        System.out.print("Enter number of Columns: ");
        int iCol = sc.nextInt();

        Pattern pobj = new Pattern();
        pobj.Pattern(iRow, iCol);
    
    }
}